easybuild/easyconfigs/i/intel/intel-2016.00.eb

```diff
@@ -0,0 +1,22 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+easyblock = "Toolchain"
+
+name = 'intel'
+version = '2016.00'
+
+homepage = 'http://software.intel.com/en-us/intel-cluster-toolkit-compiler/'
+description = """Intel Cluster Toolkit Compiler Edition provides Intel C/C++ and Fortran compilers, Intel MPI & Intel MKL."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+compver = '2016.1.150'
+gccsuff = '-GCC-4.9.3'
```

Comment:
usually, the `intel` toolchains are `GNU` based, but we're redefining `GNU` and `GCC` in other PRs so that binutils is included in both cases, so I need to take another look at that to have a full pictures here

@ocaisa: thoughts?